src/components/CommitHistory.tsx

```diff
@@ -1,49 +1,51 @@
 /* eslint-disable react-hooks/exhaustive-deps */
-import type { GitLogEntry, HydratedGitTreeObject } from "~/analyzer/model"
-import { useEffect, useMemo, useState } from "react"
-import { dateFormatLong } from "~/util"
-import commitIcon from "~/assets/commit_icon.png"
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/no-noninteractive-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
```

Comment:
Remove or add justification

Reply:
These are to silence eslint complaining about accessibility